app/services/vacancy_page_view.rb

```diff
@@ -0,0 +1,24 @@
+class VacancyPageView
+  def initialize(vacancy)
+    @vacancy = vacancy
+  end
+
+  def track
+    @vacancy.page_view_counter.increment
+  end
+
+  def persist!
+    @vacancy.total_pageviews = if @vacancy.total_pageviews.nil?
```

Comment:
We can probably just do `@vacancy.total_pageviews.to_i + @vacancy.page_view_counter.to_i` here I would have thought? As `nil.to_i` will just be `0`

Reply:
Thanks for pointing that out, it was doing my head in. I was trying to do things like `@vacancy.total_pageviews.to_i += @vacancy.page_view_counter.to_i` before 😅